packages/application-shell/src/components/authenticated/helpers.ts

```diff
@@ -0,0 +1,34 @@
+import type { ApplicationWindow } from '@commercetools-frontend/constants';
+
+import { OIDC_CLAIMS } from '../../constants';
+
+declare let window: ApplicationWindow;
+
+export const buildOidcScope = (options: {
+  projectKey?: string | null;
+}): string => {
+  const projectClaims = [];
+  if (options.projectKey) {
+    projectClaims.push(`${OIDC_CLAIMS.PROJECT_KEY}:${options.projectKey}`);
+    projectClaims.push(
+      ...(window.app.__DEVELOPMENT__?.permissions?.view ?? []).map(
+        (scope) => `${OIDC_CLAIMS.VIEW}:${scope}`
+      )
+    );
+    projectClaims.push(
+      ...(window.app.__DEVELOPMENT__?.permissions?.manage ?? []).map(
+        (scope) => `${OIDC_CLAIMS.MANAGE}:${scope}`
```

Comment:
I expected this to be `<project-key>:<oauth-scope>` not `<scope>:<oauth-scope>`.
it looks like this is by design so I could've mixed things up

Reply:
Yes the idea is to have well-defined claims:
* `project_key`
* `view`
* `manage`

Having `<project-key>:<oauth-scope>` makes it harder to parse, because you always assume that the left-hand value is the project key.